pom.xml

```diff
@@ -16,9 +16,9 @@ This is an extension module to enable Man-In-The-Middle impersonation for HTTPS.
         <github.global.server>github</github.global.server>
         <skipTests>false</skipTests>
         <!-- Versions should better be retrieved from a parent module. -->
-        <netty.version>4.0.36.Final</netty.version>
+        <netty.version>4.1.93.Final</netty.version>
```

Comment:
We should carefully test it, though

Reply:
The plan is Dev(has been there for 20h) -> Sandbox -> Line ISO in Live -> Main stack Live

Reply:
This should be in parent pom